src/sage/libs/mpmath/ext_main.pyx

```diff
@@ -848,8 +855,8 @@ cdef class Context:
                 MPF_clear(&sre)
                 MPF_clear(&sim)
                 return +unknown
-        finally:
-            sig_off()
+        except KeyboardInterrupt:
+            raise KeyboardInterrupt('Ctlr-C has been pressed')
 
```

Comment:
Typo `Ctrl-C`. 

The usual phrase seems to be this:
```
KeyboardInterrupt("Ctrl-C pressed while running %s" % self)  
```
though I see all kinds of variants in sage lib...

Reply:
fixed

Reply:
Sorry. There is no `self` here...